docs/guides/deployment/docker.md

```diff
@@ -16,10 +16,13 @@ In the [Deployments](/tutorial/deployments/) tutorial, we looked at creating con
 
 In this guide, we'll further configure the deployment so flow runs are executed in a Docker container. We'll run our Docker instance locally, but you can extend this guide to run it on remote machines.
```

Comment:
```suggestion
In this guide, we will "bake" your code directly into a Docker image. This will allow for easy storage and allow you to deploy flow code without the need for a storage block. Then, we'll configure the deployment so flow runs are executed in a Docker container based on that image. We'll run our Docker instance locally, but you can extend this guide to run it on remote machines.
```